text/0000-temp-config-settings.md

```diff
@@ -13,65 +13,368 @@ nit-picking the "gotchas" in code examples. :heart:
+the current chaos. Once again, this is a good thing because it will allow us to
+create a more consistent experience (both customer and developer) across the company.
+
+### Cost of migration
+
+This would introduce a breaking change **only** for teams that have yet to adopt
+Pando React. For teams that have already adopted Pando React, this would be a
+non-breaking change since the tooling is internal and it's just adding some deps
+and a config file.
+
+If we approach this via the CLI promoted above, it should be even easier for teams
+to start using Pando without doing any work.
 
 # Alternatives
 
-What other designs have been considered? What is the impact of not doing this?
+## Pando team specific
+
+The alternative is to continue to do what we are doing now which is basically
+doing what Panda already does so we can streamline the process and make it easier
+by just migrating to Panda.
+
+## Company wide
+
+The alternative is to continue to do what we are doing now which is basically
+allowing all teams to do whatever they want for FE styling which has led to a
+lot of unnecessary complexity and issues on different levels across the company.
 
 # Adoption strategy
 
-If we implement this proposal, how will existing Pando developers adopt it? Is
-this a breaking change? Can we write a codemod? Should we coordinate with
-other projects or libraries?
+This would (and should) require us to officially sunset any internal UI libary outside of
+Pando (Classic, ps-mui, and anything else). This would be a good thing because it would
+allow us to focus on one library and make it the best it can be for the company.
 
-# How we teach this
+This process has already begun and is in the works for the next few months via our
+ops team.
 
-What names and terminology work best for these concepts and why? How is this
-idea best presented? As a continuation of existing Pando patterns?
+# How we teach this
 
-Would the acceptance of this proposal mean the Pando documentation must be
-re-organized or altered? Does it change how Pando is taught to new developers
-at any level?
+This change shouldn't change the way we teach Pando. It should only make it easier
+to teach Pando. We would need to update the installation instructions to include
+the CLI and everything it will do for the teams.
 
-How should this feature be taught to existing Pando developers?
+Likewise, the only thing that should change on the Reference documentation is the
+customization sections which should be updated to reflect the new structure and
+show off the flexibility.
 
 # Unresolved questions
 
-Optional, but suggested for first drafts. What parts of the design are still
-TBD?
\ No newline at end of file
+- How can we continously audit the usage to ensure consistency across the company? Can we create a GH action or something similar to prevent shipping a branch that installs a new lib?
+- How can we ensure that teams are using the CLI and not just doing their own thing?
```

Comment:
Are either of these restrictions really necessary? If the benefits of these approaches are enticing enough, there should be no reason to deviate imho. Forbidding people to use other tools could be problematic. It would also be very difficult to enforce (what counts as a "new lib" in this context?). As for the first point I think it should just come down to seeing why a team might be using another library. If there is a concrete reason why a different approach is preferred (maybe a component, maybe a utility, etc.), either that feature could maybe be adapted by pando, or else perhaps it's fine for other tools to be used as long as pando remains the source of truth (i.e. via `design-tokens`/`headless-styles`).